src/components/TradeWidget/index.tsx

```diff
@@ -263,10 +200,15 @@ const TradeWidget: React.FC = () => {
   const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
 
   const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId), [fallBackNetworkId])
+  const sellInputId = TradeFormTokenId.sellToken
+  const receiveInputId = TradeFormTokenId.receiveToken
+  const priceInputId = TradeFormTokenId.price
+  const priceInverseInputId = TradeFormTokenId.priceInverse
+  const validUntilId = TradeFormTokenId.validUntil
```

Comment:
It's a good suggestion, how ever, as far as I know:
```
# this don't work, because TradeFormTokenId is a enum, not an object
const sellInputId = const {
    sellToken: sellInputId,
    receiveToken: receiveInputId,
    price: priceInputId,
    priceInverse: priceInverseInputId,
    validUntil: validUntilId,
  } = TradeFormTokenId
```

Or is there a way? @alfetopito

Reply:
You are doing `const` twice

```
const sellInputId = const {...} = TradeFormTokenId
```

Reply:
But then again, not important, just a nit pick.

Reply:
yep, the code I added had the double const. However, I think the issue is that enum is a type, cannot be destructured as far as I know.

Reply:
Got it, please disregard.